packages/react/src/stories/ic-select.stories.mdx

```diff
@@ -92,7 +94,7 @@ export const optionsWithCustomElements = [
   {
     label: "Latte",
     value: "Lat",
-    element: { component: `<ic-status-tag label="Neutral status"></ic-status-tag>`, ariaLabel: "Neutral status tag" },
+    element: { component: ReactDOMServer.renderToString(<IcStatusTag label="Neutral status tag"></IcStatusTag>), ariaLabel: "Neutral status tag" },
```

Comment:
Is it possible to add the `ReactDOMServer.renderToString` internally so the consumer only has to add `IcStatusTag` (based on your example).